frontend/packages/knative-plugin/src/components/add/event-sources/KafkaSourceSection.tsx

```diff
@@ -39,7 +48,7 @@ const KafkaSourceSection: React.FC<KafkaSourceSectionProps> = ({ title, fullWidt
             },
           ];
       placeholder = t('knative-plugin~Add bootstrap servers');
-    } else if (kafkas.loadError) {
+    } else if (isKafkasLoadError) {
```

Comment:
Since this depends on 2 resources should we also check message bellow if it is not undefined to see if error comes from Kafka source?

Reply:
`isKafkasLoadError` will true only we get err from both i.e `kafkas.loadError && kafkaconnections.loadError` so load error won't be undefined if there is err but yes it makes sense to show err from both i.e `kafkaconnections.loadError.message` as well. Have updated the code.